### DISCUSSION TITLE
my-first-post/

### DISCUSSION THREAD

**Discussion by giscus** [triccsr/triccsr.github.io#2] [Announcements]
# my-first-post/

NOTE This is a note. This is a warning. This is dangerous. TIP a tip # meow 1 2 3 4 5 6 #include <iostream> int main(){ cout<<"hello"<<endl; return 0; } # bark dfkjsdk sequenceDiagram participant Alice participant Bob Alice->>John: Hello John, how are you? loop Healthcheck John->>John: Fight against hypochondria end Note right of John: Rational thoughts prevail! John-->>Alice: Great! John->>Bob: How about you? Bob-->>John: Jolly good! Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.

http://localhost:1313/my-first-post/

**Comment by triccsr** [comment]
## My Test Comment
Is giscus working properly?
> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
>
> The **quick brown** fox jumps over the *lazy* ~dog~. 

$\forall x \in \mathbb{N}\ (x^2 \in \mathbb{N})$

```markdown
> The **quick brown** fox jumps over the *lazy* ~dog~. 
```
